Lombiq.BaseTheme/Views/Layout.cshtml

```diff
@@ -22,32 +22,32 @@
     page: this,
     new[]
     {
-        new ZoneDescriptor(Header, wrapBody: true, elementName: "header")
+        new ZoneDescriptor(Header, wrapBody: true, elementName: "header", ariaLabel: "Site")
```

Comment:
Sorry for being hyper-fixated on this, but since I've just discovered `AngleSharp.Dom.TagNames` might we use them on this page as well?

Reply:
If you have an `AngleSharp.Dom.TagNames`, everything looks like a tag name!